src/app_service/service/chat/service.nim

```diff
@@ -343,8 +343,28 @@ QtObject:
     result = ""
     try:
       var images = Json.decode(imagePathsJson, seq[string])
+      let base64JPGPrefix = "data:image/jpeg;base64,"
 
       for imagePath in images.mitems:
+
+        if imagePath.startsWith(base64JPGPrefix):
```

Comment:
This is not the image path anymore, it is image data.

Reply:
Which is overwritten after we've extracted the bytes from it.